app/views/phases/_sort_and_filter.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="grid-container padding-x-0">
+  <div class="grid-row">
+    <div class="grid-col-12 display-flex flex-column flex-align-start padding-top-105">
+      <nav aria-label="Sort and filter options" class="usa-nav" data-controller="sort-filter-menu">
+        <ul class="usa-nav__primary usa-accordion">
+          <li class="bg-primary radius-md">
+            <button
+              type="button"
+              class="usa-accordion__button usa-nav__link usa-current"
+              aria-expanded="false"
+              aria-controls="basic-nav-section-one"
+              style="&:hover { color: inherit !important; outline: none; }"
+            >
+              <span style="color: white !important;">Sort & Filter</span>
```

Comment:
do these style attributes need the !important rule? i think we can extract these custom styles to css/sass too